xlators/protocol/server/src/server-resolve.c

```diff
@@ -100,6 +100,45 @@ resolve_gfid_entry_cbk(call_frame_t *frame, void *cookie, xlator_t *this,
     return 0;
 }
 
+int
+resolve_name(call_frame_t *frame, inode_t *parent)
+{
+    server_state_t *state = NULL;
+    server_resolve_t *resolve = NULL;
+    loc_t *resolve_loc = NULL;
+    dict_t *dict = NULL;
+    xlator_t *this = frame->this;
```

Comment:
since it's only used in the error path, can be removed (and the log message would call frame->this->name